content/en/user-guide/aws/elastic-mapreduce/index.md

```diff
@@ -8,20 +8,45 @@ aliases:
   - /aws/elastic-mapreduce/
 ---
 
-LocalStack Pro allows running data analytics workloads locally via the [EMR](https://aws.amazon.com/emr) API. EMR utilizes various tools in the [Hadoop](https://hadoop.apache.org/) and [Spark](https://spark.apache.org) ecosystem, and your EMR instance is automatically configured to connect seamlessly to the LocalStack S3 API.
+## Introduction
+
+Amazon EMR (Elastic MapReduce) is a fully managed big data processing service that allows developers to effortlessly create, deploy, and manage big data applications. EMR supports various big data processing frameworks, including Hadoop MapReduce, Apache Spark, Apache Hive, and Apache Pig. Developers can leverage these frameworks and their rich ecosystem of tools and libraries to perform complex data transformations, machine learning tasks, and real-time data processing.
+
+LocalStack Pro supports EMR and allows developers to run data analytics workloads locally. EMR utilizes various tools in the [Hadoop](https://hadoop.apache.org/) and [Spark](https://spark.apache.org) ecosystem, and your EMR instance is automatically configured to connect seamlessly to LocalStack's S3 API. LocalStack also supports EMR Serverless to create applications and job runs, to run your Spark/PySpark jobs locally.
+
+The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_emr/), which provides information on the extent of EMR's integration with LocalStack.
+
+{{< alert title="Note">}}
+To utilize the EMR API, certain additional dependencies need to be downloaded from the network. This includes a Docker image of approximately 1.5 GB, which incorporates essential tools like Presto, Hive, and others. These dependencies are fetched automatically during service startup. Therefore, it's important to ensure a reliable internet connection when retrieving the dependencies for the first time.
```

Comment:
Actually, this section is a bit outdated, as we've introduced the bigdata mono container mode a while back. We can slightly rephrase to something like:
```suggestion
To utilize the EMR API, certain additional dependencies need to be downloaded from the network (including Hadoop, Hive, Spark, etc). These dependencies are fetched automatically during service startup, hence it is important to ensure a reliable internet connection when retrieving the dependencies for the first time. Alternatively, you can use one of our `*-bigdata` Docker image tags which already ship with the required libraries baked in and may provide better stability (see [here](https://docs.localstack.cloud/user-guide/ci/#ci-images) for more details).
```